chart/templates/db-deployment.yaml

```diff
@@ -62,4 +62,64 @@ spec:
         secret:
           secretName: gcloud-sql-token
 {{ toYaml .Values.defaults | indent 6 }}
-{{- end -}}
+{{- end }}
```

Comment:
Where are the sql scripts moved to? Or was that an unintentional delete?

Reply:
I have only removed on SQL script which added the gitpod testdb user.